packages/unit-testing/jest-configuration/SuiteCloudJestConfiguration.js

```diff
@@ -205,10 +205,22 @@ const CORE_STUBS = [
 		module: 'N/https/clientCertificate',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/clientCertificate/clientCertificate.js`,
 	},
+	{
+		module: 'N/keyControl',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/keyControl/keyControl.js`,
+	},
+	{
+		module: 'N/keyControl/key',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/keyControl/Key.js`,
+	},
 	{
 		module: 'N/log',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/log/log.js`,
 	},
+	{
+		module: 'N/portlet',
```

Comment:
if we are putting alphabeticly, this should go after plugin

Reply:
Reordered.